datadog_checks_dev/datadog_checks/dev/tooling/templates/check/{check_name}/manifest.json

```diff
@@ -0,0 +1,23 @@
+{{
```

Comment:
Just checking - is the double-nesting here correct?

Reply:
It is, since we use curly braces for template variables, we need to escape the actual curly braces by doubling them